src/components/MAPI/clusters/CreateCluster/patches.ts

```diff
@@ -115,17 +112,8 @@ export function withClusterControlPlaneNodesCount(count: number): ClusterPatch {
       ) {
         continue;
       }
-
-      switch (controlPlaneNode.kind) {
-        case infrav1alpha3.G8sControlPlane:
-          controlPlaneNode.spec.replicas = count;
-          break;
-        case infrav1alpha3.AWSControlPlane:
-          controlPlaneNode.spec.availabilityZones = determineRandomAZs(
-            count,
-            supportedAZs
-          );
-          break;
+      if (controlPlaneNode.kind === infrav1alpha3.G8sControlPlane) {
+        controlPlaneNode.spec.replicas = count;
```

Comment:
Nothing added, just removing the logic related to setting `availabilityZones` when HA/single masters is toggled